backend/routes.ts

```diff
@@ -349,6 +351,20 @@ const setup = ({
     retentionReasonsRouter({ prisonApi, dataComplianceApi, logError })
   )
 
+  router.use('/manage-prisoner-whereabouts/activity-results*', (req, res, next) => {
+    const { activeCaseLoadId } = req.session.userDetails
+    if (
+      activities.enabled_prisons.split(',').includes(activeCaseLoadId) &&
```

Comment:
should this be an OR? unless there a scenario where whereabouts can be part in use e.g. just for appointments?